source/includes/_daily_campaign_activity.md

```diff
@@ -0,0 +1,278 @@
+### **Daily Campaign Acticvity**
+
+### Resource Overview
+
+|Method|URI Format|
+|---|---|
+|GET|/client_reports/daily_campaign_activity/[gmaid]?[query_params]|
+
+Use GET to retrieve information for the Daily Campaign Activity report for a given advertiser.
+
+The data returned will include impressions, leads, clicks, calls, qualified_web_events, web_events, emails, and spend values organized by campaign and campaign_cycle.
+
+### Parameters
+
+When using the GET method, the results can be filtered using these parameters:
+
+|Param|Function|
+|---|---|
+|start_date|Restricts the results to those occurring on or after this date|
+|end_date|Restricts the results to those occurring on or before this date|
+|global_master_campaign_id[]|Restrict results to one or more specific campaigns. This should be a comma separated string. Ex: global_master_campaign_id[]=TEST_1,TEST_2|
+|campaign_status[]|Restrict results to all campaigns with given status values.  Allowed values are running, stopped and ended. This should be a comma separated string. Ex: campaign_status[]=running,stopped|
+|campaign_cycle|Restrict results to a single campaign cycle|
```

Comment:
Is this necessary, we don't call out cycles in the response so allowing this as a filter doesn't seem like something we want.

Reply:
[It's not supported and should be removed](https://github.com/GannettDigital/reach-analytics-reporting-service/pull/1966/files#diff-f6bde42362fb05021213f3d538f637db596738a5a6c19df9e3af1d974468ae1bR1-R9)